src/styles/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    position: relative;
+    width: 40px;
+    height: 40px;
+    left: 50px;
+
```

Comment:
For better readability, it's not recommended to add a line break between properties belonging to the same class, unless the properties are separated by categories or logical groupings.